gremlin-dotnet/src/Gremlin.Net/Process/Traversal/GraphTraversal.cs

```diff
@@ -1115,6 +1124,15 @@ private GraphTraversal(ICollection<ITraversalStrategy> traversalStrategies, Byte
             Bytecode.AddStep("label");
             return Wrap<TStart, string>(this);
         }
+        
+        /// <summary>
+        ///     Adds the length step to this <see cref="GraphTraversal{SType, EType}" />.
+        /// </summary>
+        public GraphTraversal<TStart, TEnd> Length<TNewEnd> ()
```

Comment:
In Java this returns `Number`, but isn't this usually an `int`? In that case, we could change this to:
```suggestion
        public GraphTraversal<TStart, int> Length()
```